### DISCUSSION TITLE
Need help understanding content tab size responsiveness

### DISCUSSION THREAD

**Discussion by LizThompson93** [squidfunk/mkdocs-material#7116] [Q&A]
Hello Martin!

I am using mkdocs-charts-plugin: https://github.com/timvink/mkdocs-charts-plugin
And here is the graph I am using: https://vega.github.io/vega-lite/examples/bar_percent_of_total.html

Video of problem: https://www.youtube.com/watch?v=hEygDABuNNQ

On the main level, the figure works great. Size is as expected and widening and shortening the axis. However, when the same content is place in a content tab, it becomes smaller, including text, and the biggest issue of all is that when it shrinks, it shrinks the **ENTIRE** figure. On my actual larger figures, it shrinks to the point where it becomes unreadable.

I'm just going to brainstorm some ideas but possible it's something to do with the container for tab contents, tab system being blocks, or maybe the order of operation of Vega javascript and Material's tab order but I'm not sure what I should be targeting here.

**Comment by squidfunk** [comment]
Thanks for asking. If you can provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/), somebody might be able to help you, but without that we can only guess where the problem lies. It might also be a good idea to ask the maintainers of the [mkdocs-charts-plugin](https://github.com/timvink/mkdocs-charts-plugin) or Vega for help.

**Comment by LizThompson93** [comment]
I'll revisit this issue later with a reproducible example, not critical at the moment.
